tests/unit/test_table.py

```diff
@@ -3621,11 +3617,10 @@ def test_to_dataframe_no_tqdm_no_progress_bar(self):
         user_warnings = [
             warning for warning in warned if warning.category is UserWarning
         ]
-        # With Python 3.7 and 3.8, len(user_warnings) = 3. With pandas < 1.5,
-        # pandas.ArrowDtype is not supported. We raise warnings because
-        # range columns have to be converted to object.
-        # With higher Python versions and noextra tests, len(user_warnings) = 0
-        self.assertIn(len(user_warnings), [0, 3])
+        # With pandas < 1.5 pandas.ArrowDtype is not supported. We raise warnings
+        # because range columns have to be converted to object.
+        # With Python versions > 1.5 and noextra tests, len(user_warnings) = 0
+        self.assertEqual(len(user_warnings), 0)
```

Comment:
What is the expected number of warnings with pandas < 1.5? Not sure if we still need the comments.